v3_async_wip/v3_async_wip/mqtt_client.py

```diff
@@ -121,13 +123,22 @@ def __init__(
         # Synchronization
         self.connected_cond = asyncio.Condition()
         self.disconnected_cond = asyncio.Condition()
-        self._connection_lock = ConnectionLock()
+        self._connection_lock = asyncio.Lock()
+        self._mid_tracker_lock = asyncio.Lock()
 
-        # Tasks
+        # Tasks/Futures
+        self._network_loop = None
```

Comment:
>_network_loop [](http://example.com/codeflow?start=13&length=13)

This might need a better name because of confusion with asyncio event loops.

Reply:
How about instead of changing the name of the network loop (which is the correct name), I always refer to the event loop as the event loop instead?